src/core_modules/capture-core/components/Pages/NewEnrollment/DataEntry/epics/dataEntry.epics.js

```diff
@@ -27,7 +28,7 @@ export const openNewEnrollmentInDataEntryEpic = (action$: InputObservable, store
             const state = store.value;
             const selectionsComplete = state.currentSelections.complete;
             if (!selectionsComplete) {
-                return selectionsNotCompleteOpeningNewEnrollment();
+                return of(selectionsNotCompleteOpeningNewEnrollment());
```

Comment:
I believe that from the `switchMap` we should return Observables and not single objects.
This was what flow warning us about here.